site/content/en/docs/Reference/fleetautoscaler.md

```diff
@@ -37,7 +37,7 @@ spec:
       # maximum fleet size that can be set by this FleetAutoscaler
       # required
       maxReplicas: 20
-  # [Stage:Alpha]
+  # [Stage:Beta]
```

Comment:
You will also need to feature shortcode the example:

https://agones.dev/site/docs/contribute/#within-a-page